src/chrome/content/rules/ProBoards.xml

```diff
@@ -40,6 +40,8 @@
 	<target host="storage.proboards.com" />
 	<target host="www.proboards.com" />
 
+	<target host="*.freeforums.net" />
```

Comment:
Please also add `freeforums.net` (without any subdomain) and a comment, that the wildcard is required because of user generated subdomains.